external/versions.cmake

```diff
@@ -4,28 +4,22 @@
 set(SEQUANT_TRACKED_VGCMAKEKIT_TAG 748f2499684c453109098aa06ccd03e10e915336)
 
 set(SEQUANT_TRACKED_RANGEV3_TAG 0.12.0)
-set(SEQUANT_TRACKED_RANGEV3_PREVIOUS_TAG d800a032132512a54c291ce55a2a43e0460591c7)
 
 set(SEQUANT_TRACKED_TILEDARRAY_TAG 5477174cd8e857f76d29cf6c0dcc80277393a9eb)
-set(SEQUANT_TRACKED_TILEDARRAY_PREVIOUS_TAG 5477174cd8e857f76d29cf6c0dcc80277393a9eb)
 
 set(SEQUANT_TRACKED_LIBPERM_TAG cada3e185549896203cf4d0c7f26ea22c7de428f)
-set(SEQUANT_TRACKED_LIBPERM_PREVIOUS_TAG cada3e185549896203cf4d0c7f26ea22c7de428f)
 
 set(SEQUANT_TRACKED_POLYMORPHICVARIANT_TAG 010c69786104c07c5faccffe0e99f99de5a69fd8)
-set(SEQUANT_TRACKED_POLYMORPHICVARIANT_PREVIOUS_TAG 010c69786104c07c5faccffe0e99f99de5a69fd8)
 
 set(SEQUANT_TRACKED_UTFCPP_TAG v4.0.6)
-set(SEQUANT_TRACKED_UTFCPP_PREVIOUS_TAG v4.0.6)
 
 set(SEQUANT_TRACKED_CLI11_TAG v2.5.0)
-set(SEQUANT_TRACKED_CLI11_PREVIOUS_TAG v2.5.0)
+set(SEQUANT_OLDEST_CLI11_VERSION 2)
```

Comment:
I think I'd vote for the term "minimum version" rather than "oldest version" :thinking: